demo/BiharmonicHHJ.py

```diff
@@ -3,16 +3,17 @@
 # The bilinear form a(u, v) and linear form L(v) for
 # Biharmonic equation in Hellan-Herrmann-Johnson (HHJ)
 # formulation.
-from ufl import (Coefficient, FacetNormal, FiniteElement, TestFunctions,
-                 TrialFunctions, dot, dS, ds, dx, grad, inner, jump, triangle)
+from basix.ufl import mixed_element, element
```

Comment:
Maybe `import basix.ufl as uflw` would be clearer, with `uflw.element(..)` in the code?